### PR TITLE
Cypress lesson15 Stadnichenko

### PR DESCRIPTION
**Pull request by PollySt**

### CODE REVIEW

**Comment by esegar0352** [comment]
Hi Polina!
Good work! everything works good.
Just one minor remark:
I would advise to reduce the number of strings a bit. For example, you can create one method for this assertion:

_export fucnction addUserData  (LOCATOR, USER_DATA) {
        cy.get(LOCATOR).click()
        .type(USER_DATA)
        .should("have.value", USER_DATA)
}_

and then use it:

_addUserData  (selectors.FIRST_NAME,  fakeData.USER_FIRST_NAME);_

Thus, in this test, instead of 18 lines, you get only 6.

**Comment by PollySt** [comment]
@esegar0352 thank you for advice, it's really great and helpfull)

**Comment by CyberSergo** [comment]
Hi Polina. Thank you for trusting me to check your homework. I learned a lot about Cypress for myself by checking your code (Thanks for that). The code is working, all checks were successful (except the last one), all imports and exports work correctly, variables have logical names. I would like to give you some advice:
- Don't store any JS files (except test files) in the integration folder
- don't forget to use ";"
- in faker, you can generate a random department "faker.commerce.department()".
In the last test "Verify that Action column isn't sortable" I got an error, maybe you wrote it that way on purpose, if not, please fix it

**Comment by PollySt** [comment]
@CyberSergo hey, thank for your review) I didn't see in cypress documentation any ";", so i think it's not necessary. The last column should be unsortable, but it's sortable, so it's a defect)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # lesson14_polina_stadnichenko
-You can find test file in cypress/integration/cypressTests.js
+You can find test file in cypress/integration/testingWebTables.js
```
Comment (landriiuk) [inline README.md:2]: readme file should contain information how install your frame and how run

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,5 @@ cypress/downloads/
 cypress/integration/1-getting-started/
```
Comment (landriiuk) [inline .gitignore]: cypress/fixtures/ needs for storing test data, if u have same test data u can replace there and delete this folder from the gitignore

File: cypress/integration/testingWebTables.js [hunk 3]
```diff
@@ -0,0 +1,266 @@
+import * as fakeData from "./variables.js"
+import * as sort from "./sorting.js"
+import { getInnerText, getNumbers } from "./getCellsData.js"
+import * as selectors from "./selectors.js"
+
+describe("Testing of Web Tables page", () => {
+// 1. Create a new user and verify that user was added
```
Comment (landriiuk) [inline cypress/integration/testingWebTables.js]: Pls do not use any comments in your code

File: cypress/integration/testingWebTables.js [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+        cy.get(selectors.TABLE).should("include.text", fakeData.USER_FIRST_NAME)
+        .and("include.text", fakeData.USER_LAST_NAME)
+        .and("include.text", fakeData.USER_EMAIL)
+        .and("include.text", fakeData.USER_AGE)
+        .and("include.text", fakeData.USER_SALARY)
+        .and("include.text", fakeData.USER_DEPARTMENT)
+    })
+
+    // 2. Edit user and check that each field is editable
```
Comment (landriiuk) [inline cypress/integration/testingWebTables.js]: Same here

File: package.json [hunk 5]
```diff
@@ -2,14 +2,16 @@
   "name": "function",
   "version": "1.0.0",
   "description": "aqa task",
-  "main": "./cypress/integration/cypressTests.js",
+  "main": "./cypress/integration/testingWebTables.js",
+  "type": "module",
   "scripts": {
     "test": ""
```
Comment (landriiuk) [inline package.json]: add here a few commands how to run the test

```suggestion
    "test:debug": "cypress open",
    "test:headed":"cypress run"
```

And after that u can run the tests using `npm run test:debug` or `npm run test:headed` in your terminal.

File: package.json [hunk 6]
```diff
@@ -2,14 +2,16 @@
...
   },
   "author": "AQA team",
```
Comment (landriiuk) [inline package.json]: Here should be your name

**Review by landriiuk — APPROVED** [review]

**Comment by landriiuk** [comment]
Replace files with methods for some models folder.
Integration folder should contain only test specs.

Score 9/10

**Comment by landriiuk** [comment]
@PollySt  fix comments and merge this branch into master

**Pull request merged**